README.md

```diff
@@ -36,6 +36,12 @@ needed must be installed manually on the system.
 [ansible-core]: https://docs.ansible.com/ansible-core/devel
 [Python venv doc]: https://docs.python.org/3/library/venv.html
 
+Additional `Linux`, `macOS` and `Windows with WSL2` installation instructions are available
+ in the [Installation guide][detailed installation instructions].
```

Comment:
note: this leading whitespace doesn't cause a linting violation but it should. I guess I forgot to integrate pre-commit.com here...